Updating OAuth documentation

bmignano opened the pull request:
Updating OAuth documentation to include HTPasswd and OpenID creation forms. Made minor updates to other screens based on feedback.

(Also moved Taints and Tolerations folder to fix broken link).

FYI @openshift/team-ux-review

serenamarie125 commented on web-console/futures/oauth/oauth.md:
```diff
@@ -7,37 +7,37 @@
 
-- Users can expand rows in the list in order to see the full identity provider details.
+### OpenID
+![6](img/openID.png)
```
@bmignano should asterisks be included to indicate required input?

serenamarie125 commented on web-console/futures/oauth/oauth.md:
```diff
@@ -7,37 +7,37 @@
+## IDP Forms
+- Each form will have a short description of the IDP the users have chosen to add.
+- The four common parameters to all identity providers are `name`, `challenge`, `login`, and `mappingMethod`, and will be the first items in the list.
+- Where appropriate, section headers may be used to divide the form into sub sections.
```
Is the section title being used here is something consistent with other areas of OpenShift?

bmignano commented:
@serenamarie125 Thanks for the feedback! Updated to include asterisks and additional notes about required fields where applicable. As far as I know, the only other page that has sections is the Create Silence form; I've updated the screens to look more like that form.

spadgett commented:
cc @enj 

Note that IDP name should be readonly when editing an existing IDP.

spadgett commented:
We might want "Add Claim" and "Remove Claim" for OpenID, similar to what we do for secrets:

![create image pull secret okd 2019-02-26 12-44-23](https://user-images.githubusercontent.com/1167259/53434329-5db49600-39c4-11e9-8367-9db268c0d5f5.png)

spadgett commented:
Should "Challenge" and "Login" be checkboxes instead of radios (with descriptive text)?

bmignano commented:
Hey @spadgett , thanks for the feedback! Will definitely take note of the readonly version when I design the edit view, and I think the change for challenge and login makes sense and I can update accordingly.

As for claims, would the add function allow users to add any one of these standard claims? https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims
I think those three (Name, Email, and Preferred Username) are the basic claims, so would it make sense to always show those and allow users to add additional ones, letting them specify which one with a dropdown?

bmignano commented:
Global configuration is now a separate tab again. Also removed YAML tab from IDP overview page. FYI @beanh66

cshinn commented:
Looks like a few image links are broken. Might wanna check everything over to make sure that it still works with the repo reorganization
